packages/shopify-api/lib/index.ts

```diff
@@ -32,11 +32,11 @@ export * from './flow/types';
 export interface Shopify<
   Params extends ConfigParams = ConfigParams,
   Resources extends ShopifyRestResources = ShopifyRestResources,
-  Future extends FutureFlagOptions = FutureFlagOptions,
+  _Future extends FutureFlagOptions = FutureFlagOptions,
```

Comment:
Keeping this generic type as it is referenced in `shopify-app-remix` & `shopify-app-express`